src/main/resources/paas.yaml

```diff
@@ -26,8 +26,8 @@ externalConfigDirectory: /tmp
 
 downloadConfigs: true
 
-fieldsYamlLocation: classpath://config/fields.yaml
-registersYamlLocation: classpath://config/registers.yaml
+fieldsYamlLocation: https://field.discovery.openregister.org/records.yaml
```

Comment:
Actually you're right—this should probably be configurable in `manifest.yaml` but with a sensible default.